### PR TITLE
CLM - Properties history - hide if more than 5 and show long list on modal

### PR DESCRIPTION
**Pull request by lneves12**
## What does this PR change?

CLM - Properties history - hide if more than 5 and show long list on modal

## GUI diff

Screenshot from 2019-05-02 14-52-02

Screenshot from 2019-05-02 14-52-35

- [x] **DONE**

## Documentation
- No documentation needed.

- [x] **DONE**

## Test coverage
- No tests: **add explanation**
- Unit tests were added
- Cucumber tests were added

- [x] **DONE**

## Links

Fixes #
Tracks # **add downstream PR, if any**

- [ ] **DONE**

## Changelogs

If you don't need a changelog check, please mark this checkbox:

- [x] No changelog needed

If you uncheck the checkbox after the PR is created, you will need to re-run `changelog_test` (see below)

## Re-run a test

If you need to re-run a test, please mark the related checkbox, it will be unchecked automatically once it has re-run:

- [ ] Re-run test "changelog_test"
- [ ] Re-run test "backend_unittests_pgsql"
- [ ] Re-run test "java_lint_checkstyle"		 
- [ ] Re-run test "java_pgsql_tests"		 
- [ ] Re-run test "ruby_rubocop"
- [ ] Re-run test "schema_migration_test_oracle"
- [ ] Re-run test "schema_migration_test_pgsql"		 
- [ ] Re-run test "susemanager_unittests"
- [ ] Re-run test "javascript_lint"

### CODE REVIEW

File: web/html/src/manager/content-management/shared/components/panels/properties/properties-view.js [hunk 1]
```diff
@@ -33,22 +55,29 @@ const PropertiesView= (props: Props) => {
...
+            <PropertiesHistoryEntries
+              id="resume"
+              entries={propertiesToShow.historyEntries.slice(0,NUMBER_HISTORY_ENTRIES)} />
+
+            {
+              propertiesToShow.historyEntries.length > NUMBER_HISTORY_ENTRIES &&
+              <>
+                <ModalLink
+                  id={`properties-longlist-modal-button`}
+                  text="show more"
```
Comment (ncounter) [inline web/html/src/manager/content-management/shared/components/panels/properties/properties-view.js]: I think this should be wrapped by the `t()`

File: web/html/src/manager/content-management/shared/components/panels/properties/properties-view.js [hunk 2]
```diff
@@ -33,22 +55,29 @@ const PropertiesView= (props: Props) => {
...
+                  target="properties-longlist-modal-content"
+                />
+                <Dialog
+                  id="properties-longlist-modal-content"
+                  content={
+                    <PropertiesHistoryEntries
+                      id="longlist"
+                      entries={propertiesToShow.historyEntries} />
+                  }
+                  title="Versions history"
```
Comment (ncounter) [inline web/html/src/manager/content-management/shared/components/panels/properties/properties-view.js]: `t()` function missing here as well :wink:

File: web/html/src/manager/content-management/shared/components/panels/properties/properties-view.js [hunk 3]
```diff
@@ -33,22 +55,29 @@ const PropertiesView= (props: Props) => {
         <dl className="row">
           <dt className="col-xs-2">Versions history:</dt>
```
Comment (ncounter) [inline web/html/src/manager/content-management/shared/components/panels/properties/properties-view.js]: `t()` function missing here as well and in all the previous field. I see this is something detached from this PR but maybe it's a good occasion to fix them all. :wink:

**Comment by lneves12** [comment]
> The patch LGTM, 
> There is just the `t()` function missing here and there. A part from this, the PR is ready to go. Thanks

done

**Pull request merged**